CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 lib_xud Change Log
 ==================
 
+UNRELEASED
+----------
+
+  * CHANGE:   RX_RISE_DELAY for XS2 to fix MIDI test failures
```

Comment:
We should make this more generic "RX_RISE_DELAY for XS2A based devices to resolve intermittent transmit timing issues" or similar.